src/makeRobot.js

```diff
@@ -38,7 +38,59 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return (`name: ${this.name}, `
+        + `chip version: ${this.version}, wheels: ${this.wheels}`);
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goRight(speed = 1) {
```

Comment:
```suggestion
    },
    
    get location() {
      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
    },
    
    goRight(speed = 1) {
```
it's better to add an empty line between methods

Reply:
Got it, thanks)